test/migrations/4-test4.coffee

```diff
@@ -0,0 +1,4 @@
+module.exports.id = "4-test4"
```

Comment:
I've renamed it to reflect that it's testing the whitelist matches .coffee files.